client/packages/purchasing/src/purchase_order/DetailView/ImportLines/UploadTab.tsx

```diff
@@ -73,28 +80,71 @@ export const UploadTab = ({
       },
     ]);
 
-    addCell('requestedNumberOfUnits', 'label.requested', numString =>
-      parseFloat(numString)
-    );
+    addCell('numberOfPacks', 'label.requested-packs', numString => {
+      const parsedValue = parseFloat(numString);
+      const calculatedValues = calculateUnitQuantities(status, {
+        ...importRow,
+        numberOfPacks: parsedValue,
+      });
+      Object.assign(importRow, {
```

Comment:
Processing the csv fields here instead of in the review tab since that tab's only for viewing the values